ledger-api/grpc-definitions/com/daml/ledger/api/v1/value.proto

```diff
@@ -13,7 +13,7 @@ option csharp_namespace = "Com.Daml.Ledger.Api.V1";
 
 // Encodes values that the ledger accepts as command arguments and emits as contract arguments.
 //
-// The values encoding use different four classes of non-empty strings as identifiers. Those classes are
+// The values encoding use different five classes of non-empty strings as identifiers. Those classes are
```

Comment:
We might also reconsider the text and just remove the "five" altogether, as we list the classes below so it doesn't really add much value

Reply:
Done